src/audio/AudioPlayer.ts

```diff
@@ -248,12 +252,37 @@ export class AudioPlayer extends EventEmitter {
 			throw new Error(`Cannot play a resource (${resource.name ?? 'unnamed'}) that has already ended.`);
 		}
 
+		/*
+			Attach error listeners to the stream that will propagate the error and then return to the Idle
+			state if the resource is still being used.
+		*/
```

Comment:
All other places in the code use `//` comments.
```suggestion
		// Attach error listeners to the stream that will propagate the error and then return to the Idle
		// state if the resource is still being used.
```